src/topology/metric_space/emetric_space.lean

```diff
@@ -289,3 +295,288 @@ instance emetric_space_pi [∀b, emetric_space (π b)] : emetric_space (Πb, π
     end }
 
 end pi
+
+namespace emetric
+variables {x y z : α} {ε ε₁ ε₂ : ennreal} {s : set α}
+
+/-- `eball x ε` is the set of all points `y` with `dist y x < ε` -/
+def eball (x : α) (ε : ennreal) : set α := {y | edist y x < ε}
```

Comment:
I think this can just be called `ball`; it's already namespaced.

Reply:
done